lib/pure/terminal.nim

```diff
@@ -467,14 +467,15 @@ proc resetAttributes*(f: File) =
     f.write(ansiResetCode)
 
 type
-  Style* = enum         ## different styles for text output
+  Style* = enum          ## different styles for text output
     styleBright = 1,     ## bright text
     styleDim,            ## dim text
-    styleUnknown,        ## unknown
+    styleItalic,         ## italic (supported on many terminals)
```

Comment:
Hm, perhaps I'm being nitpicky, but would a better word be "most" rather than"many"? "most" carries the implicit message that there are still exceptional cases.

Reply:
An even better message would be to specify precisely which terminals are *not* supported.

Reply:
@Varriount Makes sense. I'll make that change.. but see below. 

@dom96 I think that it's not worthwhile to know the supported ANSI codes for all terminals out there.. there might be dozens of terminals out there. It's a very time consuming exercise with little to no benefit. Additionally, even if the terminal supports these ANSI codes, but a user has tweaked the termcaps, certain ANSI codes can stop working. 

Tomorrow if a terminal adds support for italics, there's no way for Nim to keep track of that.  I think that the user using these ANSI codes should know what their terminal supports. 

Now I am thinking if we should even include that disclaimer in the source code if it's not Nim's responsibility to track this. Instead, in the commit message, I can say that xterm is one the popular terminals supporting these. 

What do you guys think?

Reply:
@Varriount @dom96 Please review the updated PR.. I removed the "many terminals" reference.